### ISSUE TITLE
About the setting of training loss

### ISSUE THREAD

**Issue by daiyixiang666** [facebookresearch/vggt#244]
Thanks author for your great work!
I wonder if you use the gradient_loss during training or just the normal reg and conf loss. Thanks a lot

**Comment by jytime** [comment]
Hi, as discussed in the paper, we use gradient_loss together with others during training, which is also the setting in default.yaml now.

**Comment by daiyixiang666** [comment]
Thanks !
